code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -15,13 +15,20 @@
 	var/wall_mounted = 0 //never solid (You can always pass over it)
 	var/health = 100
 	var/lastbang
+	var/can_weld_shut = 1
 	var/max_mob_size = MOB_SIZE_HUMAN //Biggest mob_size accepted by the container
 	var/mob_storage_capacity = 3 // how many human sized mob/living can fit together inside a closet.
-	var/storage_capacity = 30 //This is so that someone can't pack hundreds of items in a locker/crate
-							  //then open it in a populated area to crash clients.
+	var/storage_capacity = 30 //This is so that someone can't pack hundreds of items in a locker/crate then open it in a populated area to crash clients.
+	var/cutting_tool = /obj/item/weapon/weldingtool
+	var/open_sound = 'sound/machines/click.ogg'
+	var/cutting_sound = 'sound/items/Welder.ogg'
+	var/material_drop = /obj/item/stack/sheet/metal
 
 /obj/structure/closet/New()
 	..()
+	if(type == /obj/structure/closet && ticker && ticker.current_state == GAME_STATE_PREGAME && prob(20)) //only roundstart generic closets

```

Comment:
It's only generic closets, the grey ones with nothing on them that are found almost exclusively in maintenance. All crates are still crates, and all special lockers are still those lockers.

I can do that later and revert this bit, it's just nothing screams "This will conflict for the next six weeks" quite like a map change that touches a lot of tiles